src/compute/src/render/reduce.rs

```diff
@@ -230,10 +230,27 @@ where
                 errors.push(errs);
                 MzArrangement::RowRow(output)
             }
-            ReducePlan::Basic(BasicPlan::Single(index, aggr)) => {
-                let (output, errs) = self
-                    .build_basic_aggregate(collection, index, &aggr, true, key_arity, mfp_after);
-                errors.push(errs.expect("validation should have occurred as it was requested"));
+            ReducePlan::Basic(BasicPlan::Single(SingleBasicPlan {
+                index,
+                expr,
+                fused_unnest_list,
+            })) => {
+                // Note that we skip validating for negative diffs when we have a fused unnest list,
+                // because this is already a CPU-intensive situation due to the non-incrementalness
+                // of window functions.
+                let validating = !fused_unnest_list;
```

Comment:
Could you add an issue to enable validation with fused list unnest at a later time? Thank you